packages/webapp/src/routes/land/add-landowner-individual.js

```diff
@@ -38,12 +39,14 @@ const handlers = {
 
     const firstNameError = validateFirstLastName(firstName, 'first name', 'firstNameId')
     const lastNameError = validateFirstLastName(lastName, 'last name', 'lastNameId')
+    const middleNameError = validateLengthOfCharsLessThan50(middleName, 'middle name', 'middleNameId')
 
     if (!isEmpty(firstNameError) || !isEmpty(lastNameError)) {
```

Comment:
Need to add the check for `!isEmpty(middleNameError)` here too

Reply:
middle name can be empty, we only want this check for first and last name.

Reply:
I get it now :), I will add it

Reply:
done